### PR TITLE
feat(capacity): allow p.g tx's to pay with capacity

### PR DESCRIPTION
**Pull request by enddynayn**
# Goal
Allow for the following private-graph calls to pay transaction fees with capacity:
- apply_item_actions
- upsert_page
- delete_page
- apply_item_actions_with_signature
- upsert_page_with_signature
- delete_page_with_signature

The fees for these transactions in Capacity are fixed to this point in time by
not allowing new benchmarks to affect the weight of these transactions.

issue-1266

### CODE REVIEW

File: pallets/frequency-tx-payment/src/capacity_stable_weights.rs [hunk 1]
```diff
@@ -61,6 +61,13 @@ pub trait WeightInfo {
 	// Messages
 	fn add_onchain_message(n: u32, ) -> Weight;
 	fn add_ipfs_message() -> Weight;
+	// Stateful-storage
```
Comment (aramikm) [inline pallets/frequency-tx-payment/src/capacity_stable_weights.rs:64]: This seems to be a bit tricky, how would you know what stable weights are correct? Weights can not be assumed to be stable and I'm not sure what is the consequences for  relying on something like this

File: pallets/stateful-storage/src/tests.rs [hunk 2]
```diff
@@ -2640,3 +2641,22 @@ fn signature_replay_on_deleted_page_check() {
 		));
 	})
 }
+
+#[test]
+fn test_sum_add_actions_bytes() {
```
Comment (aramikm) [inline pallets/stateful-storage/src/tests.rs:2646]: I think you'll need to rebase from main since your test files structure is changed

**Review by aramikm — APPROVED** [review]
The implementation looks good but the underlying assumptions might be concerning.

**Review by shannonwells — APPROVED** [review]
Looks good to me, I'm still assuming we're intentionally setting capacity fees based on current weight calculations.  I think we should do a last run on Jenkins and update all these before the release though.

File: pallets/frequency-tx-payment/src/capacity_stable_weights.rs (hunk 1 above)
Comment (enddynayn) [reply pallets/frequency-tx-payment/src/capacity_stable_weights.rs:64]: I'll be working on putting something to address some of your questions tomorrow.

File: pallets/stateful-storage/src/tests.rs (hunk 2 above)
Comment (enddynayn) [reply pallets/stateful-storage/src/tests.rs:2646]: Sounds good. I will rebase the capacity-development branch and update this.

**Pull request merged**